tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -120,6 +121,14 @@ export const checkSharee = async (args: ShareArgs): Promise<void> => {
   }
 }
 
+export const isAcceptedSharePresent = async (
+  page: Page,
+  resource: string,
+  owner: string
+): Promise<boolean> => {
+  return await acceptedShareExists({ page, resource, owner })
```

Comment:
Wrapping a fn. with another fn? :thinking:  This will only add depth.

Reply:
thanks i forget to remove while doing code cleanup

Reply:
index.ts file index all the fn in action.ts